packages/@apphosting/adapter-nextjs/src/bin/build.ts

```diff
@@ -50,21 +48,22 @@ const generateBundleYaml = async () => {
     const redirects = manifest.redirects.filter(it => !it.internal).map(it => ({...it, regex: undefined}));
     const beforeFileRewrites = Array.isArray(manifest.rewrites) ? manifest.rewrites : manifest.rewrites?.beforeFiles || [];
     const rewrites = beforeFileRewrites.map(it => ({...it, regex: undefined}));
+    const outputBundleDirectory = dirname(outputBundlePath);
     await writeFile(outputBundlePath, yamlStringify({
         headers, 
         redirects, 
         rewrites,
-        runCommand: `node ${serverFilePath}`,
-        neededDirs: [appHostingOutputDirectory],
-        staticAssets: [appHostingStaticDirectory, appHostingPublicDirectory],
+        runCommand: `node ${normalize(relative(outputBundleDirectory, serverFilePath))}`,
```

Comment:
I checked with Wei, and his buildpack step want to use paths relative to the base directory, so suggesting changing `outputBundleDirectory` to `cwd`

Reply:
can do. including run command? or just the directories?

Reply:
including the run command